modules/react-native-status/ios/RCTStatus/RCTStatus.h

```diff
@@ -4,6 +4,8 @@
 #import <JavaScriptCore/JavaScriptCore.h>
 #import "Jail.h"
 
+#define NSLog(s, ...) do { NSLog(s, ##__VA_ARGS__); TFLog(s, ##__VA_ARGS__); } while (0)
```

Comment:
We need to make sure that that doesn't go to any of the app store builds though.
@rasom is the best person to talk about that :)

Reply:
In app store builds, won't `TESTFAIRY_ENABLED` be set to 0 (from `.env.prod`)?

Reply:
yeah, I was expecting something like
```objc
#if defined(TESTFAIRY_ENABLED) && TESTFAIRY_ENABLED
   #define NSLog(s, ...)....
#endif
```

Reply:
Yeah, easy enough to do that. I have to confess I assumed `TFLog` would be a no-op if `TESTFAIRY_ENABLED` wasn't set to `1` (this commit was meant to serve for testing first and not really PR material). Good catch!

Reply:
Even if it is, I like being explicit 🤓